packages/types/src/multiDomain.ts

```diff
@@ -1,6 +1,7 @@
 import type { ClerkOptions } from './clerk';
 
 type DOMAIN = string | ((url: URL) => string);
+type PROXY_URL = string | ((url: URL) => string);
```

Comment:
❓ why should this be a different type and not have a `StringOrFn` (or whatever name) and use it both for `DOMAIN` / `PROXY_URL` ?

Reply:
Honestly I couldn't think of the name of the type 😂 so just copied it. I'll use `StringOrFn` thanks for the inspiration @dimkl